src/app/chat-list-item/chat-list-item.component.html

```diff
@@ -0,0 +1,27 @@
+<button
+  *ngIf="chatMetadata"
+  mat-list-item
+  class="h-auto my-2"
+  (click)="onClick()"
+>
+  <div class="container">
+    <div class="row">
+      <div class="col-auto p-0">
+        <img
+          class="rounded-circle"
+          height="56px"
+          [src]="chatMetadata.groupImageUrl"
+        />
+      </div>
+      <div class="col">
+        <span>{{ chatMetadata.groupName }}</span> <span>Time</span>
+        <p
+          *ngIf="lastMessage && users[lastMessage.uid]"
+          class="pt-2 text-muted"
+        >
+          {{ users[lastMessage.uid].displayName }}: {{ lastMessage.message }}
```

Comment:
Since the image upload as a chat is in the main branch do you want to check whether the tag is text or image so if it is an image don't display the whole Url instead just say display 
"Ryan: shared an image" instead of 
"Ryan: https://firebase..."

Reply:
Yeah when main is merged into this branch, that should be accomodated.